src/Thentos/Util.hs

```diff
@@ -77,3 +83,13 @@ readsPrecEnumBoundedShow _ s = f [minBound..]
         (s0, s1) -> if s0 == s' then [(x, s1)] else f xs
       where
         s' = show x
+
+
```

Comment:
what about this?

    $ git grep readsPrecEnumBoundedShow
    src/Thentos/Backend/Api/Adhocracy3.hs:    readsPrec = readsPrecEnumBoundedShow
    src/Thentos/Backend/Api/Adhocracy3.hs:    readsPrec = readsPrecEnumBoundedShow